jupyter-book/glossary.md

```diff
@@ -1,41 +1,56 @@
+
 Algorithm
 Algorithms
     A pre-defined set of instructions to solve a problem.
 
 AnnData
 AnnDatas
-    A Python package for annotated data matrices. The primary data structure used in the scverse ecosystem.
+    A Python package for annotated data matrices.
+    The primary data structure used in the scverse ecosystem.
+
+Backtrace
+backtrace
+    In the context of sequence alignment, a backtrace refers to the step-by-step reconstruction of the alignment path that produces the optimal alignment score.
+    This path shows how the sequences are aligned, including matches, mismatches, insertions, deletions, and gaps.
 
 Barcode
 Barcodes
 Bar code
 Bar codes
-    Short DNA barcode fragments ("tags") that are used to identify reads originating from the same cell. Reads are later grouped by their barcode during raw data processing steps.
+    Short DNA barcode fragments ("tags") that are used to identify reads originating from the same cell.
+    Reads are later grouped by their barcode during raw data processing steps.
 
 Batch effect
-    Technical confounding factors in an experiment that cause dataset distribution shifts. Usually lead to inaccurate conclusions if the causes of the batch effects are correlated with outcomes of interest in an experiment and should be accounted for (usually removed).
+    Technical confounding factors in an experiment that cause dataset distribution shifts.
+    Usually lead to inaccurate conclusions if the causes of the batch effects are correlated with outcomes of interest in an experiment and should be accounted for (usually removed).
 
 Benchmark
     An (independent) comparison of performance of several tools with respect to pre-defined metrics.
 
 Bulk RNA sequencing
-    Contrary to single-cell sequencing, bulk sequencing measures the average expression values of several cells. Therefore, resolution is lost, but bulk sequencing is usually cheaper, less laborious and faster to analyze.
+    Contrary to single-cell sequencing, bulk sequencing measures the average expression values of several cells.
+    Therefore, resolution is lost, but bulk sequencing is usually cheaper, less laborious and faster to analyze.
 
 Cell
-    The fundamental unit of life. Consists of cytoplasm enclosed within a membrane that contains many biomolecules such as proteins and nucleic acids. Cells acquire specific functions, transition to cell types, divide, communicate and keep the organism going. Learning about the structure, activity and communication of cells helps deciphering biology.
+    The fundamental unit of life.
+    Consists of cytoplasm enclosed within a membrane that contains many biomolecules such as proteins and nucleic acids.
+    Cells acquire specific functions, transition to cell types, divide, communicate and keep the organism going.
+    Learning about the structure, activity and communication of cells helps deciphering biology.
```

Comment:
Maybe make this more precise? Not just `deciphering biology`. Maybe we can mention 1-3 use cases here such as gene expression dynamics, cellular trajectories and lineages, ...